### PR TITLE
Clean-up: Remove unused private variables (Issue #1621, #1622, #1624)

### PR DESCRIPTION
**Pull request by talih0**
Pull request removes unused private variables (Issue #1621, #1622, #1624)
One unused function mag_squared in feedforward_agc_cc_impl.cc was also removed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,8 @@
 *~
 *.pyc
 *.pyo
+cscope.files
+cscope.in.out
+cscope.out
+cscope.po.out
```
[the remark this answers is not in the document]
Comment (talih0) [reply .gitignore]: Thanks, this is good to know

**Comment by marcusmueller** [comment]
so, @talih0, personally, I think this looks pretty good; I can't claim I have reviewed in-depth right now, but most of these affect  blocks that are no longer actively worked on, so I guess you're doing no harm there.

If I was submitting this PR, I'd try to split it into multiple commits, namely one for each gr-filter, gr-digital, gr-blocks to make exactly that kind of verification easier. Now, you're really only removing basic types (and in one case, a "dead end" method), so this is fine by me.

I'd put this in the 3.8+ release regime, maybe I can smuggle it into the 3.8 release itself.

One thing that we should probably get out of the way (since you look like someone who's willing to continuously contribute) is copyright. I'll hand over to @bhilburn for that!

**Comment by bhilburn** [comment]
@talih0 - In order to get this merged, we'll need a CLA from you! Can you e-mail me <bhilburn@gnuradio.org>, and we'll get the process started? Thanks! `=)`

**Comment by dkozel** [comment]
Thanks @talih0!
I'd meant to finish my sentence in my review with:
_I think it's fine to remove_ all these unused variables and people can add them back in when they actually do something.

**Comment by talih0** [comment]
Thanks! 
@bhilburn, I just sent you an email for the CLA

**Comment by marcusmueller** [comment]
@bhilburn @talih0 update on the CLA?

**Comment by talih0** [comment]
@bhilburn @marcusmueller, I just forwarded you the signed CLA

**Comment by marcusmueller** [comment]
Awesome! Thanks, @talih0, and my sincerest apologies it took this long!

**Pull request merged**